src/react-components/object-list.js

```diff
@@ -112,9 +113,10 @@ export default class ObjectList extends Component {
 
   domForEntity(el, i) {
     return (
-      <div
+      <button
+        aria-label="Show Object Info Modal"
```

Comment:
I would probably use "Panel" instead of "Modal". I don't know if the technical difference matters here, and some users might not be familiar with the concept of a modal.

Reply:
Good call!